src/db-service/shopping_cart.js

```diff
@@ -10,7 +10,7 @@ exports.createCart = async (name, email) => {
   return result[0];
 };
 
-exports.checkIfHasCart = async email => {
+exports.checkIfHasCart = async (email) => {
   const result = await knex("shopping_carts")
     .select("id")
     .where({ owner_email: email });
```

Comment:
Nit: Fits on one line

Reply:
Off the top of my head, this seems feasible, but that would make MemoryRetainer more sophisticated than what it currently is - it is now only for the heap snapshot integration, moving memory management bits there may require a bit more thoughts into this...(the general idea of making it more powerful sounds cool though, and probably can help with nodejs/diagnostics#155)

Reply:
Please focus on readability for others to understand

Reply:
test

Reply:
d

Reply:
a

Reply:
1

Reply:
3

Reply:
4

Reply:
test2

Reply:
te3

Reply:
test

Reply:
test

Reply:
test

Reply:
1

Reply:
3

Reply:
3

Reply:
4

Reply:
5

Reply:
123

Reply:
4

Reply:
5

Reply:
3

Reply:
1

Reply:
112

Reply:
241

Reply:
23143

Reply:
32312341234

Reply:
1234

Reply:
last test 🗡

Reply:
not so last test...

Reply:
💯

Reply:
huh

Reply:
asdfqwer

Reply:
2134